www/_data/tools.yml

```diff
@@ -1,9 +1,10 @@
-# -   name: Example Tool
-#     image: cordova.png
-#     url: http://example.com
+# -   name: GapDebug
+#     image: gapdebug-logo100.png
+#     url: https://www.genuitec.com/products/gapdebug/
 #     description: >
-#         A description that can
-#         span multiple lines.
+#         GapDebug is a free tool for debugging and managing hybrid-mobile Android and iOS apps. 
+#         With Drag and drop App Installation, Instant debugging, and full cross-platform support, 
+#         GapDebug changes how you view debugging for good.
```

Comment:
Hey Dmitry,

I did the change like you asked. Crossing my fingers!!

Thanks!!



On Wed, Oct 7, 2015 at 9:47 PM, Dmitry Blotsky <notifications@github.com>
wrote:

> In www/_data/tools.yml
> <https://github.com/apache/cordova-docs/pull/369#discussion_r41471386>:
>
> >  #     description: >
> > -#         A description that can
> > -#         span multiple lines.
> > +#         GapDebug is a free tool for debugging and managing hybrid-mobile Android and iOS apps.
> > +#         With Drag and drop App Installation, Instant debugging, and full cross-platform support,
> > +#         GapDebug changes how you view debugging for good.
>
> No problem at all! Don't worry, a PR is made precisely so that you can
> safely change things and nothing will be affected until it's merged. I'll
> take a look again whenever you have a moment to update it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/cordova-docs/pull/369/files#r41471386>.
>